paper/paper.Rmd

```diff
@@ -1,35 +1,42 @@
 ---
-title: "datawizard: An R Package for Easy Data Wrangling"
+title: "datawizard: An R Package for Easy Data Wrangling and Statistical Transformations"
```

Comment:
I would call it "Data Preparation", it encompasses wrangling + transfo + implies that it's pre-analysis

Reply:
Good point. Done!